Misc/largestRange.java

```diff
@@ -0,0 +1,46 @@
+import java.util.*;
+public class largestRange
+{
+    // Finds the length of longest occurring consecutive numbers range in an array
+    public static int longestRange(int[] nums)
+    {
+        int longestRange = 0;
+        HashMap<Integer,Boolean> num = new HashMap<>(); // Stores a mapping of a number to whether the current number is part of a particular consecutive range or not
```

Comment:
You would face a lot of problem while formatting this comment. Rather than using `// ` try something like this - 
`/** Stores a mapping of a number to whether the current number {NEW_LINE} *  is part of a particular consecutive range or not.
   */` 
These javadoc comments are formated better by google-java-format, as well as prettier.